### PR TITLE
TIQR-0000:  refactor sample app style, and icons

### PR DESCRIPTION
**Pull request by sarahachem**

### CODE REVIEW

File: app/build.gradle.kts [hunk 1]
```diff
@@ -11,14 +11,14 @@ android {
     buildToolsVersion = libs.versions.android.buildTools.get()
 
     defaultConfig {
-        applicationId = "org.tiqr.authenticator"
-        versionCode = 30
-        versionName = "4.0.0"
+        applicationId = "org.nl.sample"
```
Comment (dima-ko) [inline app/build.gradle.kts]: maybe org.tiqr.core or something like that?

File: app/src/main/AndroidManifest.xml [hunk 2]
```diff
@@ -17,14 +17,12 @@
     </queries>
 
     <application
-        android:name="org.tiqr.TiqrApplication"
+        android:name="org.nl.SampleApplication"
```
Comment (dima-ko) [inline app/src/main/AndroidManifest.xml]: We should decide if we really need it in this project. If we use our own applications in tiqr and eduID, and don't extend it, maybe remove it?

File: app/build.gradle.kts (hunk 1 above)
Comment (welsinga) [reply app/build.gradle.kts]: Doesn't this effect the notification flow?

File: app/src/main/AndroidManifest.xml (hunk 2 above)
Comment (welsinga) [reply app/src/main/AndroidManifest.xml]: also org.nl is wrong, it would be org.sample

Comment (sarahachem) [reply app/src/main/AndroidManifest.xml]: @pmeulen thought it's a good idea to keep it as a playground app, but i can remove it also no problem

Comment (dima-ko) [reply app/src/main/AndroidManifest.xml]: @sarahachem ah, yeah, you're right, keep it

**Review by welsinga — APPROVED** [review]

File: app/build.gradle.kts (hunk 1 above)
Comment (dima-ko) [reply app/build.gradle.kts]: @sarahachem maybe net.surf.core ? or nl.surfnet.core. ask customer plz

Comment (welsinga) [reply app/build.gradle.kts]: No, it's still an organisation. So domain should be org and surfnet isn't the name anymore, it's surf.

Comment (welsinga) [reply app/build.gradle.kts]: @sarahachem correction, url is https://www.surf.nl/ so name should be nl.surf -> nl.surf.sample

**Review by welsinga — APPROVED** [review]

**Pull request merged**